Makefile

```diff
@@ -86,7 +91,7 @@ $(CLIENT_TARGET): $(K8S_GEN_DEPS)
 	--output-package "$(GO_PKG)/pkg/client"
 
 LISTER_TARGET := pkg/client/listers/monitoring/v1/prometheus.go
-$(LISTER_TARGET): $(K8S_GEN_DEPS)
+$(LISTER_TARGET): $(K8S_GEN_DEPS	)
```

Comment:
nit: i think the tab character is not necessary here.

Reply:
Yes, noticed too late that I had accidentally added it for some reason. Fixed in followup commit.